### ISSUE TITLE
Unwanted scrolling if there are few elements

### ISSUE THREAD

**Issue by Katarn** [DesarrolloAntonio/Shiori-Android-Client#31]
If the total number of elements is less than one screen, you need to disable scrolling.

https://github.com/DesarrolloAntonio/Shiori-Android-Client/assets/6339978/5435a999-7794-4022-9fdf-9650773d9fc6

**Comment by DesarrolloAntonio** [comment]
Fixed in v1.18.1

**Comment by Katarn** [comment]
I don't see any difference from how it was before.

**Comment by DesarrolloAntonio** [comment]
> If the total number of elements is less than one screen, you need to disable scrolling.

I cannot disable scrolling because it would result in losing the pull-to-refresh functionality.

**Comment by Katarn** [comment]
But these are two different functions, aren’t they? Scrolling should be disabled and the list will have a fixed position. At the same time, when you pull it down, the update icon should appear.

**Comment by DesarrolloAntonio** [comment]
You're right! Pull-to-refresh and scrolling are separate functions. The scrolling behavior you're seeing is Android's native implementation, which automatically handles content that fits within one screen.
